### ISSUE TITLE
Natural Language Processing

### ISSUE THREAD

**Issue by SchwarzKatzeZwei** [SchwarzKatzeZwei/Keras-Code-samples#2]
- [ ] Text classification from scratch
- [ ] Sequence to sequence learning for performing number addition
- [ ] Bidirectional LSTM on IMDB
- [ ] Character-level recurrent sequence-to-sequence model
- [ ] End-to-end Masked Language Modeling with BERT
- [ ] Named Entity Recognition using Transformers
- [ ] English-to-Spanish translation with a sequence-to-sequence Transformer
- [ ] Natural language image search with a Dual Encoder
- [ ] Using pre-trained word embeddings
- [ ] Semantic Similarity with BERT
- [ ] Text classification with Switch Transformer
- [ ] Text classification with Transformer
- [ ] Text Extraction with BERT
